scripts/repo.js

```diff
@@ -177,9 +174,14 @@ async function buildDocs (pathToDocs) {
 
     await fs.writeFile(path.resolve(pathToDocs, 'mkdocs.yml'), mkdocsTemplate({language, themeDir, toc}));
     const pathToBuildDocsTo = path.resolve(__dirname, '..', 'docs', language);
-    await exec('mkdocs', ['build', '--site-dir', pathToBuildDocsTo], {
-      cwd: pathToDocs,
-    });
+    try {
+      await exec('mkdocs', ['build', '--site-dir', pathToBuildDocsTo], {
+        cwd: pathToDocs,
+      });
+    } catch (e) {
+      console.log('Could not build', e.message);
+      console.log('Reason:', e.stderr);
```

Comment:
do we ignore the exception deliberately?

Reply:
yeah, should probably still throw an error here right if we can't build?